domain-models-runtime/src/main/java/org/folio/rest/tools/utils/TenantInit.java

```diff
@@ -0,0 +1,92 @@
+package org.folio.rest.tools.utils;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.client.TenantClient;
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.jaxrs.model.TenantJob;
+
+public final class TenantInit {
+  private static final Logger log = LogManager.getLogger(TenantInit.class);
+
+  private TenantInit() {
+    throw new UnsupportedOperationException("Cannot instantiate utility class.");
+  }
+
+  /**
+   * Perform tenant purge, wait for result and delete job.
+   * @param client Tenant Client.
+   * @param waitMs number of milliseconds to wait for job completion (0=no wait).
+   * @return async result.
+   */
+  public static Future<Void> purge(TenantClient client, int waitMs) {
+    TenantAttributes at = new TenantAttributes().withPurge(true);
+    return exec(client, at, waitMs);
+  }
```

Comment:
Curious, is `module id` not needed for `purge`?

Reply:
Not really.